lib/rules/max-empty-lines/README.md

```diff
@@ -58,3 +58,45 @@ a {}
 
 b {}
 ```
+
+## Optional secondary options
+
+### `ignore: ["comments"]`
+
+Only enforce the adjacent empty lines limit for lines that are not comments.
+
+For example, with `2` adjacent empty lines:
+
+The following patterns are considered warnings:
+
+```css
+/* horse */
+a {}
+
+
+
+b {}
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+/**
+ * Call me Ishmael.
+ *
+ *
+ *
+ * Some years ago -- never mind how log precisely -- ...
+ */
+```
```

Comment:
This example is almost exactly as the next example. Can you delete one or change it for something different, please? E. g. put this comment into `a {}`.